_posts/2015/2015-11-16-carbon-automobiles-bebop-fashion.md

```diff
@@ -0,0 +1,148 @@
+
+> “I don’t want to fight old battles, I want to fight new ones.”
+> <cite>Satya Nadella — Microsoft CEO</cite>
+
+We are seeing this happen again in the auto industry with Tesla [licensing all their patents for use](http://www.teslamotors.com/blog/all-our-patent-are-belong-you).
+
+> “We believe that Tesla, other companies making electric cars, and the world would all benefit from a common, rapidly-evolving technology platform.”
+
+> “Technology leadership is not defined by patents, which history has repeatedly shown to be small protection indeed against a determined competitor, but rather by the ability of a company to attract and motivate the world’s most talented engineers. We believe that applying the open source philosophy to our patents will strengthen rather than diminish Tesla’s position in this regard.” –Elon Musk
+
+Carbon and automobiles show what can happen when we create open platforms. What happens when everyone then builds on these open platforms?
+
+## Bebop
+
+Bebop is a form of Jazz that started to make an appearance in the 1940s, fifty years into the evolution of jazz.
+
+Like carbon and automobiles, Jazz is also a stacked platform, built on African rhythms, European harmonic accompaniment, the blue notes of racial tension, and a surplus of post-civil-war military instruments. Jazz is known for being incredibly competitive. In Jam sessions musicians take turns improvising, trying to one-up each other and then mimicking the sound they just heard.
+
+The evolution of Jazz is marked by the repeated pattern of someone playing in a slightly different style, and then everyone else imitating it. If you were a trumpet player, and you heard Louis Armstrong, then you wanted to play like him, because that's the greatest trumpet you've ever heard.
+
+Charlie Parker was instrumental in the creation of the style of Jazz called Bebop. There’s an anecdote that the reason he played in this style that was because he didn’t think white musicians could copy him. But eventually they did, because Bebop is what most of us think of when we think Jazz.
+
+Charlie’s innovation was created on the open platform of Jazz. When you chose to build on an open platform, you get a leg up. Charlie didn’t have to popularize Jazz or the blues scales or invent the saxophone. His innovation was taking things that already existed and combined them in interesting ways. Bebop may or may not have happened if Charlie Parker wasn’t involved, but once it existed, it wasn’t terribly difficult for other musicians with similar backgrounds to incorporate his innovation.
+
+We are seeing this same thing happen now in software. Even the most innovate of companies, like Apple, are at most a few years ahead of their competition in certain ways.
```

Comment:
innovate->innovative

obvious other point could be worked in there like

Even the most innovative and proprietary companies, like Apple, are at most a few years of ahead of their competition in certain ways and make large investments in open source.

Or maybe that's muddying two separate points.